gitlab4j-models/src/main/java/org/gitlab4j/api/models/Project.java

```diff
@@ -789,7 +828,7 @@ public void setBuildCoverageRegex(String buildCoverageRegex) {
 
     public Project withBuildCoverageRegex(String buildCoverageRegex) {
         this.buildCoverageRegex = buildCoverageRegex;
-        return this;
+        return (this);
```

Comment:
I don't see why we need those brackets.

I would just do this:

```suggestion
        return this;
```